contracts/StandardAssetRegistry.sol

```diff
@@ -198,14 +198,28 @@ contract StandardAssetRegistry is AssetRegistryStorage, IAssetRegistry {
     _;
   }
 
+  modifier senderIsDefined(address sender) {
+    require(sender != 0);
+    _;
```

Comment:
@eordano check this pls

Reply:
looks good. Could be more generic but I like that it's very semantic.
`isSenderDefined()`?